uk.co.compucorp.civicrm.tasksassignments/tests/phpunit/bootstrap.php

```diff
@@ -9,8 +9,17 @@
 require_once DRUPAL_ROOT . '/includes/bootstrap.inc';
 drupal_bootstrap(DRUPAL_BOOTSTRAP_FULL);
 
+addCiviHRExtensionToIncludePath('/uk.co.compucorp.civicrm.hrcore/');
+
 require_once 'TasksassignmentsTestTrait.php';
 
+function addCiviHRExtensionToIncludePath($extensionDirectory) {
+  $civicrmRoot = realpath($GLOBALS['civicrm_root']);
+  $civiHRDir = $civicrmRoot . '/tools/extensions/civihr';
+  $extensionDir = $civiHRDir . $extensionDirectory;
+  set_include_path($extensionDir . PATH_SEPARATOR . get_include_path());
+}
+
```

Comment:
By removing the usage of the HRCore fabricator, this won't be necessary anymore. As a rule of thumb, we shouldn't include things like this. If you need a class from another extension, please install it using the `setupHeadless` method

Reply:
I agree it's not perfect, but I don't think the solution is to install hrcore whenever we need to use test related classes from it, that just seems heavy-handed. This was a compromise, but as you mentioned we can remove it now.